tests/base/vectorization_11.cc

```diff
@@ -52,20 +52,25 @@ struct Tester<double>
   test()
   {
```

Comment:
Is there a reason to rename the file?

Reply:
The test used to only test `VectorizedArray::make_vectorized_array`. Now it also tests the constructor. This is the reason for the name change.

Reply:
OK.